public/api/docs/v1/swagger.yaml

```diff
@@ -923,7 +923,7 @@ paths:
               "$ref": "#/components/schemas/ParticipantChangeScheduleRequest"
         required: true
 servers:
-- url: https://npq-registration-separation-web.teacherservices.cloud/
+- url: http://0.0.0.0:3000
```

Comment:
Minor comment
Instead of "http://0.0.0.0:3000", should we just have the variable name:
```yaml
url: ACCESS_EXTERNAL_DOMAIN
```
That way the gsub is more specific.

Reply:
Good suggestion, but it would need to be a valid URL or rswag complains; it could be `http://swagger.url` or something, but I figured it was easier to leave it as the development URL so it just works in that environment (otherwise we'd need to have more logic to set it explicitly as dev doesn't have the `ACCESS_EXTERNAL_DOMAIN` env var)